swagger.yaml

```diff
@@ -250,328 +234,328 @@ paths:
+                "code": "SBDMBA-------------------------100",
+                "description": "Ladies' Downhill Run 1",
+                "start": "2016-11-25T09:45:00Z",
+                "end": "2016-11-26T14:20:00Z",
+                "meta": {
+                  "medal": "none"
+                },
+                "type": {
+                  "code": "ATH",
+                  "description": "Individuals"
+                },
+                "scheduleStatus": {
+                  "code": "DELAYED",
+                  "description": "Delayed",
+                  "notes": "Competition has not started and start is late (no new time)"
+                },
+                "phaseType": {
+                  "code": "0",
+                  "description": "Technical Meeting"
+                },
+                "venue": {
+                  "code": "ASJ",
+                  "description": "Alpensia Ski Jumping Ctr",
+                  "city": "Gangneung"
+                },
+                "games": {
+                  "code": "OWG2018",
+                  "description": "Olympic Winter Games PyeongChang 2018"
+                },
+                "discipline": {
+                  "code": "ALP",
+                  "description": "Alpine Skiing"
+                },
+                "event": {
+                  "code": "ALPWDH------09110-----",
+                  "description": "Women's Downhill, Standing",
+                  "meta": {
+                    "teamEvent": true
+                  }
+                },
+                "gender": {
+                  "code": "M",
+                  "description": "Male"
+                },
+                "phase": {
+                  "code": "ALPMAC----------------FNL-",
+                  "description": "Men's Alpine Combined"
+                },
+                "entrant": [
+                  {
+                    "startOrder": 2,
+                    "sortOrder": 2,
+                    "competitor": {
+                      "code": "CURXTEAM2---DEN01",
+                      "description": "Denmark",
+                      "type": "T",
+                      "organisation": {
+                        "code": "DEN",
+                        "description": "Denmark"
+                      }
+                    }
+                  },
+                  {
+                    "startOrder": 1,
+                    "sortOrder": 1,
+                    "competitor": {
+                      "code": "CURXTEAM2---BEL01",
+                      "description": "Belgium",
+                      "type": "T",
+                      "organisation": {
+                        "code": "BEL",
+                        "description": "Belgium"
+                      }
+                    }
+                  }
+                ],
+                "link": [{
+                  "rel": "self",
+                  "href": "https://olympics.api.press.net/v3/games/OWG2018/unit/SBDMBA----------------------------"
```

Comment:
Are these URLs correct? Shouldnt they be `pamedia.io` domains?

Reply:
Good spot